charts/karmada/values.yaml

```diff
@@ -340,20 +340,9 @@ apiServer:
   ## @param apiServer.hostNetwork
   ## "true" means using hostNetwork
   ## "false" means normal network
-  hostNetwork: true
```

Comment:
Is there a reason why use the host network before?

Reply:
I don't know either.

Reply:
The same as artifacts.

Reply:
I'm not sure if the default value of hostNetwork should be false.

Reply:
1. When I install multiple Karmada instances in the same k8s cluster, the apiserver pod keeps pending.

2. The apiserver itself does not require a host network.

Reply:
Please update README in charts. And then let's move forward.